<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CtrlTrainDiagram</class>
 <widget class="QWidget" name="CtrlTrainDiagram">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="TrainCtrl_label_train">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>441</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(133, 133, 133);
border:2px;
border-radius:10px;
padding:2px 4px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelline1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>2</width>
     <height>83</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelline1_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>2</width>
     <height>83</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelline1_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>2</width>
     <height>83</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelline1_4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>2</width>
     <height>83</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelline1_5">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>2</width>
     <height>83</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="TrainCtrl_label_MC1Door1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TrainCtrl_label_MC1Door2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TrainCtrl_label_MC1Door3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TrainCtrl_label_MC1Door4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TrainCtrl_label_MC1Door8">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TrainCtrl_label_MC1Door7">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>7</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TrainCtrl_label_MC1Door6">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>6</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TrainCtrl_label_MC1Door5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TrainCtrl_label_MC2Door7">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TrainCtrl_label_MC2Door6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TrainCtrl_label_MC2Door5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TrainCtrl_label_MC2Door8">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TrainCtrl_label_MC2Door2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TrainCtrl_label_MC2Door3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TrainCtrl_label_MC2Door4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>60</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TrainCtrl_label_MC2Door1">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>120</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelbackground">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>58</y>
     <width>445</width>
     <height>85</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 0);
border:2px;
background-color: rgb(0, 0, 0);
border-radius:10px;
padding:2px 4px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 75 18px;
</string>
   </property>
   <property name="text">
    <string>MC1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>150</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 75 18px;
</string>
   </property>
   <property name="text">
    <string>TP1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 75 18px;
</string>
   </property>
   <property name="text">
    <string>M1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>150</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 75 18px;
</string>
   </property>
   <property name="text">
    <string>TP2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 75 18px;
</string>
   </property>
   <property name="text">
    <string>MC2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 75 18px;
</string>
   </property>
   <property name="text">
    <string>M2</string>
   </property>
  </widget>
  <widget class="QLabel" name="TrainCtrl_label_MC1Brake">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:red;
border:2px;
border-radius:10px;
padding:2px 4px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="TrainCtrl_label_TC1Brake">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
border:2px;
border-radius:10px;
padding:2px 4px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="TrainCtrl_label_M1Brake">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
border:2px;
border-radius:10px;
padding:2px 4px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="TrainCtrl_label_M2Brake">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
border:2px;
border-radius:10px;
padding:2px 4px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="TrainCtrl_label_TC2Brake">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
border:2px;
border-radius:10px;
padding:2px 4px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="TrainCtrl_label_MC2Brake">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
border:2px;
border-radius:10px;
padding:2px 4px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="TrainCtrl_label_NET">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>9</y>
     <width>425</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="TrainCtrl_label_MC1active">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/images/MC1Active.bmp);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="TrainCtrl_label_MC2active">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/images/MC2Active.bmp);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="TrainCtrl_label_MC1MRLS">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/images/warning.bmp);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TrainCtrl_label_M1MRLS">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/images/warning.bmp);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TrainCtrl_label_M2MRLS">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/images/warning.bmp);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TrainCtrl_label_MC2MRLS">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/images/warning.bmp);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>labelbackground</zorder>
  <zorder>TrainCtrl_label_train</zorder>
  <zorder>labelline1</zorder>
  <zorder>labelline1_2</zorder>
  <zorder>labelline1_3</zorder>
  <zorder>labelline1_4</zorder>
  <zorder>labelline1_5</zorder>
  <zorder>TrainCtrl_label_MC1Door1</zorder>
  <zorder>TrainCtrl_label_MC1Door2</zorder>
  <zorder>TrainCtrl_label_MC1Door3</zorder>
  <zorder>TrainCtrl_label_MC1Door4</zorder>
  <zorder>TrainCtrl_label_MC1Door8</zorder>
  <zorder>TrainCtrl_label_MC1Door7</zorder>
  <zorder>TrainCtrl_label_MC1Door6</zorder>
  <zorder>TrainCtrl_label_MC1Door5</zorder>
  <zorder>TrainCtrl_label_MC2Door7</zorder>
  <zorder>TrainCtrl_label_MC2Door6</zorder>
  <zorder>TrainCtrl_label_MC2Door5</zorder>
  <zorder>TrainCtrl_label_MC2Door8</zorder>
  <zorder>TrainCtrl_label_MC2Door2</zorder>
  <zorder>TrainCtrl_label_MC2Door3</zorder>
  <zorder>TrainCtrl_label_MC2Door4</zorder>
  <zorder>TrainCtrl_label_MC2Door1</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>TrainCtrl_label_MC1Brake</zorder>
  <zorder>TrainCtrl_label_TC1Brake</zorder>
  <zorder>TrainCtrl_label_M1Brake</zorder>
  <zorder>TrainCtrl_label_M2Brake</zorder>
  <zorder>TrainCtrl_label_TC2Brake</zorder>
  <zorder>TrainCtrl_label_MC2Brake</zorder>
  <zorder>TrainCtrl_label_NET</zorder>
  <zorder>TrainCtrl_label_MC1active</zorder>
  <zorder>TrainCtrl_label_MC2active</zorder>
  <zorder>TrainCtrl_label_MC1MRLS</zorder>
  <zorder>TrainCtrl_label_M1MRLS</zorder>
  <zorder>TrainCtrl_label_M2MRLS</zorder>
  <zorder>TrainCtrl_label_MC2MRLS</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
